src/applications/vaos/containers/DateTimeRequestPage.jsx

```diff
@@ -67,56 +27,58 @@ export class DateTimeRequestPage extends React.Component {
 
   componentDidMount() {
     focusElement('h1.vads-u-font-size--h2');
-    this.props.openFormPage(pageKey, uiSchema, initialSchema);
     document.title = `${pageTitle} | Veterans Affairs`;
   }
 
-  componentDidUpdate(prevProps, prevState) {
-    if (!prevState.validationError && this.state.validationError?.length > 0) {
+  componentDidUpdate(prevProps) {
+    if (
+      !prevProps.data?.calendarData?.error &&
+      this.props.data?.calendarData?.error?.length > 0
+    ) {
       scrollAndFocus('.usa-input-error-message');
     }
-
-    const prevSelectedSlotsCount =
-      prevProps.data.calendarData?.selectedDates?.length || 0;
-    const newSelectedSlotsCount =
-      this.props.data.calendarData?.selectedDates?.length || 0;
-
-    if (prevSelectedSlotsCount !== newSelectedSlotsCount) {
-      this.validate();
-    }
   }
 
+  getOptionsByDate = () => [
+    {
+      value: 'AM',
+      label: 'AM',
+    },
+    {
+      value: 'PM',
+      label: 'PM',
+    },
+  ];
+
   goBack = () => {
+    this.props.clearCalendarData();
     if (!this.state.validationError) {
       this.props.routeToPreviousAppointmentPage(this.props.router, pageKey);
     }
   };
 
   goForward = () => {
-    this.validate();
-    if (this.userSelectedSlot()) {
+    this.props.validateCalendar(pageKey);
```

Comment:
I think some of these back/forward changes could follow what I did in https://github.com/department-of-veterans-affairs/vets-website/pull/11736 to line up better with the way validation works on other pages.